src/main/scala/io/citrine/lolo/transformers/Standardizer.scala

```diff
@@ -0,0 +1,166 @@
+package io.citrine.lolo.transformers
+
+import io.citrine.lolo.{Learner, Model, PredictionResult, TrainingResult}
+
+/**
+  * Created by maxhutch on 2/19/17.
+  */
+class Standardizer(baseLearner: Learner) extends Learner() {
+  override var hypers: Map[String, Any] = Map()
+
+  override def setHypers(moreHypers: Map[String, Any]): this.type = {
+    baseLearner.setHypers(moreHypers)
+    super.setHypers(moreHypers)
+  }
+
+  /**
+    * Train a model
+    *
+    * @param trainingData to train on
+    * @param weights      for the training rows, if applicable
+    * @return training result containing a model
+    */
+  override def train(trainingData: Seq[(Vector[Any], Any)], weights: Option[Seq[Double]]): StandardizerTrainingResult = {
+    val rep = trainingData.head._1
+    val inputTrans = Standardizer.getMultiStandardization(trainingData.map(_._1))
+    val outputTrans = trainingData.head._2 match {
+      case x: Double => Some(Standardizer.getStandardization(trainingData.map(_._2.asInstanceOf[Double])))
+      case x: Any => None
+    }
+
+    val (inputs, labels) = trainingData.unzip
+    val standardTrainingData = Standardizer.applyStandardization(inputs, inputTrans).zip(Standardizer.applyStandardization(labels, outputTrans))
+    val baseTrainingResult = baseLearner.train(standardTrainingData)
+
+    new StandardizerTrainingResult(baseTrainingResult, Seq(outputTrans) ++ inputTrans, hypers)
+  }
+}
+
+class StandardizerTrainingResult(
+                                  baseTrainingResult: TrainingResult,
+                                  trans: Seq[Option[(Double, Double)]],
```

Comment:
You know me and variable names. Coming from outside the codebase, it is not clear that a Seq[Option[(Double, Double)]] called "trans" is a linear transformation with parameters (a, b): x -> (x - a) / b